StokedSportsWebApi/wwwroot/index.js

```diff
@@ -1,18 +1,55 @@
-﻿
-async function sendAuthorizedRequestAsync(apiUrl, methodType, data) {
```

Comment:
why was line 2 removed? Just to make sure this wasn't an accident.

Reply:
Hmm, I''m not sure.  I think it was by accident.